### PR TITLE
Fix bad icon path in Desktop file

### PR DESCRIPTION
**Pull request by jkonecny12**
Icon should be used from /usr/share/pixmaps not from application folder.

### CODE REVIEW

**Comment by jkonecny12** [comment]
The same here. Do you plan to merge it?

**Comment by DinCahill** [comment]
Hi. I know this has been sitting around for ages, but the desktop file has been changed, so this needs rebasing/merging master.

**Comment by jkonecny12** [comment]
**Rebased**

**Comment by qparis** [comment]
I don't think this commit will work with official .deb packages

**Comment by jkonecny12** [comment]
It won't work because of Debian doesn't use this structure or because PlayOnLinux deb package doesn't use this structure?

**Comment by qparis** [comment]
PlayOnLinux deb package does not

**Comment by jkonecny12** [comment]
I think it should be fixed by the clean way, so to use pixmap directory. I'll look on your building scripts when I found some free time if I can't fix it there. Do you agree on this solution.

File: etc/PlayOnLinux.desktop [hunk 1]
```diff
@@ -4,5 +4,5 @@ Name=PlayOnLinux
 Comment=Front-end application for the wine
 Type=Application
 Exec=playonlinux %F
-Icon=/usr/share/playonlinux/etc/playonlinux.png
+Icon=/usr/share/pixmaps/playonlinux.png
```
Comment (AsciiWolf) [inline etc/PlayOnLinux.desktop]: Is there any reason for using a hardcoded path instead of just "playonlinux" for the icon name?

Comment (AsciiWolf) [inline etc/PlayOnLinux.desktop]: (If the icon is in _/usr/share/pixmaps_ or other standard location.)

**Comment by AsciiWolf** [comment]
@qparis It will work.

`$ locate playonlinux.png`
`/usr/share/pixmaps/playonlinux.png`
`/usr/share/playonlinux/etc/playonlinux.png`

(On Debian Unstable machine with latest PlayOnLinux package installed.)

File: etc/PlayOnLinux.desktop (hunk 1 above)
Comment (jkonecny12) [inline etc/PlayOnLinux.desktop]: I just modified the old version I think it could work with just `playonlinux`.

**Comment by AsciiWolf** [comment]
@qparis Will this pull request get merged?

**Pull request merged**